Safari/Info.plist

```diff
@@ -61,6 +61,7 @@
 				<string>*.xkit.info</string>
 				<string>new-xkit.github.io</string>
 				<string>*.xkitcs.com</string>
+				<string>localhost:31337</string>
```

Comment:
probably don't need this in the release version, can we just add it to dev extensions when we need it?